From ee7824d4c1207d128ba8238a12fb910953e6daa3 Mon Sep 17 00:00:00 2001
From: Tianshu Qiu <tian.shu.qiu@intel.com>
Date: Tue, 15 May 2018 14:15:12 +0800
Subject: [PATCH 025/129] media: intel-ipu4: imx319: Remove kernel version
 checking

Message for Open Source:
Remove kernel version checking

Message for Internal:
[Issue/Feature] Kernel version checking is not necessary as
the driver here is only used on kernel 4.14 and later.
[Root Cause/Changes] Remove kernel version checking

Change-Id: I73f4843ebf6df307731cd866949c4a6777219b67
Tracked-On: #JICCCN-171
Signed-off-by: Tianshu Qiu <tian.shu.qiu@intel.com>
Signed-off-by: Meng Wei <wei.meng@intel.com>
---
 drivers/media/i2c/imx319.c | 8 --------
 1 file changed, 8 deletions(-)

diff --git a/drivers/media/i2c/imx319.c b/drivers/media/i2c/imx319.c
index 75a717b..0821e00 100644
--- a/drivers/media/i2c/imx319.c
+++ b/drivers/media/i2c/imx319.c
@@ -2430,19 +2430,11 @@ static int imx319_probe(struct i2c_client *client,
 	imx319->sd.internal_ops = &imx319_internal_ops;
 	imx319->sd.flags |= V4L2_SUBDEV_FL_HAS_DEVNODE;
 	imx319->sd.entity.ops = &imx319_subdev_entity_ops;
-#if LINUX_VERSION_CODE > KERNEL_VERSION(4, 4, 0)
 	imx319->sd.entity.function = MEDIA_ENT_F_CAM_SENSOR;
-#else
-	imx319->sd.entity.type = MEDIA_ENT_T_V4L2_SUBDEV_SENSOR;
-#endif
 
 	/* Initialize source pad */
 	imx319->pad.flags = MEDIA_PAD_FL_SOURCE;
-#if LINUX_VERSION_CODE > KERNEL_VERSION(4, 4, 0)
 	ret = media_entity_pads_init(&imx319->sd.entity, 1, &imx319->pad);
-#else
-	ret = media_entity_init(&imx319->sd.entity, 1, &imx319->pad, 0);
-#endif
 	if (ret) {
 		dev_err(&client->dev, "%s failed:%d\n", __func__, ret);
 		goto error_handler_free;
-- 
1.9.1

